### DISCUSSION TITLE
Vue Swiper (v9) triggers hundreds of events with 'autoplay' option

### DISCUSSION THREAD

**Discussion by galimovkirill** [nolimits4web/swiper#6437] [Q&A] [answered]
After Swiper migration from 8 to 9 version I have an issue with spamming about 100 events per second. It has been revealed from Vue Devtools
![image](https://user-images.githubusercontent.com/55671052/221576358-7a7d2ce8-83a9-46ef-bf24-9afa6a2c4a89.png)

Code example:
```
<swiper
    :loop="true"
    :speed="500"
    :space-between="0"
    :pause-on-mouse-enter="true"
    :autoplay="{ delay: 4000, disableOnInteraction: false, pauseOnMouseEnter: true }"
    :pagination="{ clickable: true }"
    :modules="modules"
    wrapper-tag="ul"
    :threshold="16"
    @swiper="onSwiper"
>
        <swiper-slide
            v-for="promo in promoList"
            :key="promo.id"
            tag="li"
            @click.prevent="selectOffer(promo)"
        >
            <img
                :alt="promo.name"
                :src="getPromoImage(promo)"
            >
        </swiper-slide>
</swiper>
```

**Comment by nolimits4web** [comment, marked as answer]
it is supposed to work in this way, as it triggers this event every frame

**Reply by galimovkirill** [reply]
> it is supposed to work in this way, as it triggers this event every frame

Does it affect on performance? Swiper v8 autoplay triggers only once (based on vue devtools) with same environment and has same behavior
![image](https://user-images.githubusercontent.com/55671052/221579089-cee6e77a-8c01-4889-9b8d-41286acaa979.png)

**Comment by nolimits4web** [comment]
> Does it affect on performance?

No, it doesn't

**Reply by galimovkirill** [reply]
> > Does it affect on performance?
> 
> No, it doesn't

Can you explain different behavior in devtools on each version?
